passwordstrengthmeter/handler.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright © 2015-2018 Aeneas Rekkas <aeneas+oss@aeneas.io>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author		Aeneas Rekkas <aeneas+oss@aeneas.io>
+ * @copyright 	2015-2018 Aeneas Rekkas <aeneas+oss@aeneas.io>
+ * @license 	Apache-2.0
+ */
+
+package passwordstrengthmeter
+
+import (
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/ory/herodot"
+
+	"github.com/pkg/errors"
+
+	"github.com/ory/x/jsonx"
+
+	"github.com/nbutton23/zxcvbn-go"
+)
+
+const (
+	// PasswordStrengthPath is the path where you can check strength of password
+	PasswordStrengthPath = "/password/strength/meter"
+)
+
+// RoutesToObserve returns a string of all the available routes of this module.
+func RoutesToObserve() []string {
```

Comment:
This is nowhere used. Please don't just copy source files from one destination to another and replace the names.